nvbios/bios.h

```diff
@@ -298,13 +299,13 @@ enum envy_bios_gpio_tag {
 
 	/* 0x30 seen, output or neg input [G200, GF100], *twice*... and sometimes in lots of copies */
 
-	/* 0x34 seen, neg input [G96, GT216, GT218, GK104] SPEC NVIO on G96, uses unk41_line */
+	ENVY_BIOS_GPIO_THERM_ALERT		= 0x34,
 
 	ENVY_BIOS_GPIO_FAN_TACH			= 0x3d,
 
 	ENVY_BIOS_GPIO_SLI_SENSE_0		= 0x40, /* XXX: uses unk40_0, unk41_4, unk41_line */
```

Comment:
"Thermal Alert: Interrupt input from external thermal device. Indicates that the device needs to be serviced" that's in the doc